src/main/scala/io/github/evis/scalafix/maven/plugin/params/ProjectDependenciesParam.scala

```diff
@@ -1,18 +1,23 @@
 package io.github.evis.scalafix.maven.plugin.params
 
-import org.apache.maven.artifact.Artifact
+import java.util.jar.JarFile
+
+import scala.util.Try
 
-import scala.collection.JavaConverters._
+import org.apache.maven.artifact.Artifact
 
 object ProjectDependenciesParam {
 
-  def apply(projectDependencies: java.util.Set[Artifact]): MojoParam = {
-    val deps = projectDependencies.asScala.toList
+  def apply(projectDependencies: Iterable[Artifact]): MojoParam = {
+    val deps = projectDependencies.toList
     val scalaLibrary =
       deps.find { dep =>
         dep.getGroupId == "org.scala-lang" && dep.getArtifactId == "scala-library"
       }
-    _.withClasspath(deps.map(_.getFile.toPath))
-      .withScalaVersion(scalaLibrary.map(_.getVersion))
+    _.withClasspath(deps.flatMap { dep =>
+      val file = dep.getFile
+      val ok = !file.isFile || Try(new JarFile(file)).isSuccess
```

Comment:
Why do we need this validation? When a dependency can be something that's not a jar file?